pkg/services/ngalert/models/instance.go

```diff
@@ -78,11 +70,7 @@ type ListAlertInstancesQuery struct {
 
 // ValidateAlertInstance validates that the alert instance contains an alert rule id,
 // and state.
-func ValidateAlertInstance(alertInstance *AlertInstance) error {
-	if alertInstance == nil {
-		return fmt.Errorf("alert instance is invalid because it is nil")
-	}
-
+func ValidateAlertInstance(alertInstance AlertInstance) error {
```

Comment:
do we really need validation for this struct? the only consumer and producer of this data - state manager.

Reply:
I haven't changed this function, save changing the type from a struct pointer to a struct. Is this blocking a merge, or do you have other feedback?